tools/reindex-index-data

```diff
@@ -11,6 +11,20 @@ sys.path.append('/usr/lib/archivematica/archivematicaCommon')
 import elasticSearchFunctions
 
 
+def index_iterator(records, index, doc_type, action='index'):
+    """
+    Creates an iterator which yields dicts suitable for use with Elasticsearch's bulk operation API.
+    Used by `reindex` to transform a list of _source records into actions ready for indexing.
+    """
+    for record in records:
+        yield {
+            '_op_type': action,
+            '_index': index,
+            '_type': doc_type,
+            'doc': record,
```

Comment:
It's optional; ES will generate a new ID if necessary. Same behaviour when indexing a record using the existing API, I believe.